### ISSUE TITLE
High CPU Usage with Streamlabs Widgets

### ISSUE THREAD

**Issue by JeffHoogland** [bazukas/obs-linuxbrowser#53]
I am using this plugin with OBS 21.0.1 on Ubuntu 16.04. I have 4 difference instances of it active in my scene which are capturing four different URLs from streamlabs. I have a fairly reasonable i7 -4790 processor.

When I watch my task manager while streaming there is a browser-subproc that is constantly using 40%~ of one of my processor cores even when nothing is changing on the URLs being captured. This is more CPU than the rest of OBS is using. When the URLs start seeing some action, there are other browser-subproc instances that start eating CPU as well, some of them using over a full core for a short period of time.

Is there anything I can do to change / improve performance? It causes my system to start to bog in a noticeable way on occasion.

**Comment by grimmy** [comment]
Alerts seems to be the worst offender here.. Although the tip jar does make a noticable difference to.  I'm running version 0.3.1 of the plugin.

**Comment by NexAdn** [comment]
Maybe #59 can help you if you figure out how to stop some of the scripts that are running.

**Comment by JeffHoogland** [comment]
@NexAdn do I need to recompile the plugin to get --enable-gpu-compositing to work or is it a setting somewhere in the plugin in OBS?

**Comment by NexAdn** [comment]
Should be fine if you pass these args using the provided source properties window.

**Comment by JeffHoogland** [comment]
Ahh so I just put them here then? 
![image](https://user-images.githubusercontent.com/3093360/49116942-bd37c400-f264-11e8-9ebf-fbd0909379cb.png)

**Comment by NexAdn** [comment]
Exactly. I've also noticed that using many keyframe animations can cause high CPU usage.
